dockerfiles/che/Dockerfile

```diff
@@ -5,8 +5,10 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #
-FROM openjdk:8u242-jre-slim
+FROM alpine:3.11
+RUN apk --no-cache add "openjdk11-jre=11.0.5_p10-r0"  bash
```

Comment:
- when I build che-server image like this, it has 273MB
 - there is jre-headless package in alpine `openjdk11-jre-headless=11.0.5_p10-r0`, which produces 262MB image
 - with `adoptopenjdk/openjdk11:alpine-jre` resulting image has 233MB, but it's only x86_64
 - with `openjdk:11-jre-slim` -> 292 MB and does not have s390x
 - ubi-minimal is ~350MB

I would change package to `openjdk11-jre-headless=11.0.5_p10-r0`. I haven't tested it though...

Reply:
reverted to Debian based image